cmd/completion.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright © 2021 Parris Lucas
```

Comment:
You'll have to replace this with the one [present](https://github.com/kubernetes-sigs/depstat/blob/main/main.go#L1-L15) in rest of the files.

Reply:
Done